data/i18n/en/en.toml

```diff
@@ -202,6 +202,14 @@ other = "Objectives"
 [options_heading]
 other = "Options"
 
+[outdated_page_message]
+other = "Kubernetes authors consider this page to be outdated because REASON. We suggest that you use more recent resources."
+[outdated_page_reason]
+other = "it is more than a year old"
```

Comment:
I would have this as a single entry:
```toml
[deprecation_warning_blog_over_one_year_old]
other = "The Kubernetes project considers this article to be outdated because it is more than one year old. Check that the information in the page has not become incorrect since its publication."
```
(and then adjust the layouts to match).

Reply:
I combined the message and reason as you suggested, but used the key `outdated_blog_over_one_year_old` instead. Does that work for you?

I updated the other key to match: `outdated_blog_title`.